fs/copy_test.go

```diff
@@ -46,6 +46,45 @@ func TestCopyDirectoryWithLocalSymlink(t *testing.T) {
 	}
 }
 
+// TestCopyWithLargeFile tests copying a file whose size > 2^32 bytes.
+func TestCopyWithLargeFile(t *testing.T) {
```

Comment:
I'll consider this for a follow up, but skipping on `testing.Short()` may be a good idea